zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/AlertServiceImpl.java

```diff
@@ -482,6 +537,43 @@ protected Alert buildAlert(final Integer alertId, final Set<String> entities, fi
         return alert;
     }
 
+    protected Alert buildAlertWithoutEntities(final Integer alertId, final long entities, final AlertDefinition definition) {
+        final List<LastCheckResult> checkResults = new LinkedList<>();
+
+        if (entities > 0) {
+            final List<ResponseHolder<String, String>> results = new LinkedList<>();
+
+            //getAlertEntityData(alertId, entities, results);
+
+            // process results
+            for (final ResponseHolder<String, String> entry : results) {
```

Comment:
`results ` is always empty here. Please remove this `for` loop

Reply:
updated and pushed code